ed_perception_msgs/package.xml

```diff
@@ -10,10 +13,11 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
+  <depend>std_msgs</depend>
+  
   <build_depend>message_generation</build_depend>
-  <build_depend>std_msgs</build_depend>
 
-  <run_depend>message_runtime</run_depend>
-  <run_depend>std_msgs</run_depend>
+  <build_export_depend>message_runtime</build_export_depend>
```

Comment:
Isn't this an execution dependency (is in one of the [examples](http://wiki.ros.org/catkin/package.xml))